ci/deps/travis-36-cov.yaml

```diff
@@ -40,7 +40,7 @@ dependencies:
   - xlwt
   # universal
   - pytest>=4.0.2
```

Comment:
you need pytest<5.0.0; I *thought* we had added this (its on all the other 3.6 builds)